apollo-normalized-cache-sqlite/api/android/apollo-normalized-cache-sqlite.api

```diff
@@ -1,77 +1,10 @@
-public abstract interface class com/apollographql/apollo3/cache/normalized/sql/ApolloDatabase : com/squareup/sqldelight/Transacter {
-	public static final field Companion Lcom/apollographql/apollo3/cache/normalized/sql/ApolloDatabase$Companion;
-	public abstract fun getCacheQueries ()Lcom/apollographql/apollo3/cache/normalized/sql/CacheQueries;
-}
-
-public final class com/apollographql/apollo3/cache/normalized/sql/ApolloDatabase$Companion {
-	public final fun getSchema ()Lcom/squareup/sqldelight/db/SqlDriver$Schema;
-	public final fun invoke (Lcom/squareup/sqldelight/db/SqlDriver;)Lcom/apollographql/apollo3/cache/normalized/sql/ApolloDatabase;
-}
-
```

Comment:
I moved all the SQLDelight generated classes into an `internal` package. Ideally we would generate them internal but it's not possible just yet, see https://github.com/cashapp/sqldelight/issues/1333

This is a breaking change for any code calling this from outside the module but as this is an implementation detail, I think we can afford this.